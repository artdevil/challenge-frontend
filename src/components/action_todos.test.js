import React from 'react';
import { shallow } from 'enzyme';

import ActionTodos from './action_todos';

describe('<ActionTodos />', () => {
  it('shallow action_todos renders without crashing', () => {
    const wrapper = shallow(<ActionTodos />);
    expect(wrapper).toMatchSnapshot();
  });
});
