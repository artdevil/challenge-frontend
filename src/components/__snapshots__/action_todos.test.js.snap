// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ActionTodos /> shallow action_todos renders without crashing 1`] = `
<Apollo(ActionTodos)
  clearCompleted={[Function]}
/>
`;
